Doc/tools/extensions/pyspecific.py

```diff
@@ -21,7 +21,6 @@
 from docutils.utils import new_document
 from sphinx import addnodes
 from sphinx.builders import Builder
-from sphinx.domains.python import PyFunction, PyMethod
 from sphinx.errors import NoUri
```

Comment:
These are used:
```pytb
  File "/home/runner/work/cpython/cpython/Doc/tools/extensions/pyspecific.py", line 342, in <module>
    class PyDecoratorFunction(PyDecoratorMixin, PyFunction):
                                                ^^^^^^^^^^
NameError: name 'PyFunction' is not defined
```


```suggestion
from sphinx.domains.python import PyFunction, PyMethod
from sphinx.errors import NoUri
```

Reply:
Oops, merge error--will resolve when I'm back at my computer!

Thanks for catching it :)

A